src/lib/components/Treebank/Sentence/Sentence.js

```diff
@@ -15,8 +17,16 @@ const findWord = (wordId, sentence) => (
 );
 
 const WrappedSentence = ({
```

Comment:
This library is built using something called React Contexts. A Context is a way of providing values to child components (and grandchild components, great-grandchild components, etc.) without explicitly passing them in. It's sort of like having a variable be available in all functions within its scope:

```javascript
// This is like a Context Provider
const someVariable = 5;

function () {
  function () {
    function () {
       // This is like a Context Receiver
       // It has access to the `someVariable` variable without it being passed in as an argument
       console.log(someVariable);
    }
  }
}
```

---

In the same way, the `<Sentence>` component is providing variables within its scope. This PR is adding the `highlighted` variable to the Context:

```jsx
<Sentence highlighted={["1", "2"]}>
  <div>
    {/* This component has access to `highlighted` even though it is not passed in directly */}
    <Text />
  </div>
</Sentence>
```

---

The reason that this library is set up this way, is it allows the user to structure their application however they want while still having access to all of the components of this library. For example, they could put the `<Text>` below the `<Graph>`, have two separate `<Graph>`s, whatever.